### PR TITLE
Generate one image with all display controllers

### PR DESCRIPTION
**Pull request by klutchell**
Avoid creating individual images for each display
controller and instead just create one image and require
that `FBCP_DISPLAY` is set.

This commit also includes support for the `dtoverlay`
display controller which is just the generic rpi-fbcp
that requires `BALENA_HOST_CONFIG_dtoverlay` for drivers.

Change-type: major
Signed-off-by: Kyle Harding <kyle@balena.io>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,46 +1,71 @@
...
+RUN echo "target_link_libraries(fbcp pthread bcm_host vchiq_arm vcos atomic)" >> CMakeLists.txt
 
-RUN sed -i '212 s/^/\/\//' config.h
+WORKDIR /usr/src/rpi-fbcp/build
 
-RUN sed -i '$d' CMakeLists.txt
+RUN cmake .. && make && mv fbcp /usr/bin/fbcp-dtoverlay
 
-RUN echo "target_link_libraries(fbcp-ili9341 pthread bcm_host vchiq_arm vcos atomic)" >> CMakeLists.txt
+RUN ldd /usr/bin/fbcp-dtoverlay
```
Comment (rahul-thakoor) [inline Dockerfile]: hey @klutchell I'm not sure if we need this line?

other than that LGTM

**Review by rahul-thakoor — COMMENTED** [review]
do we need to run ldd on rpi-fbcp binary?

**Comment by rahul-thakoor** [comment]
I tested on pi zero W. it works.

File: Dockerfile (hunk 1 above)
Comment (klutchell) [reply Dockerfile]: Oops! I'll remove it!

**Review by rahul-thakoor — APPROVED** [review]
LGTM. Thanks @klutchell

**Pull request merged**